ohapi/api.py

```diff
@@ -190,15 +182,41 @@ def message(subject, message, access_token, all_members=False,
         base_url, '/api/direct-sharing/project/message/?{}'.format(
             urlparse.urlencode({'access_token': access_token})))
     if not(all_members) and not(project_member_ids):
-        return requests.post(url, data={'subject': subject,
-                                        'message': message})
+        response = requests.post(url, data={'subject': subject,
+                                            'message': message})
+        handle_error(response)
+        return response
     elif all_members and project_member_ids:
         raise ValueError(
             "One (and only one) of the following must be specified: "
             "project_members_id or all_members is set to True.")
     else:
-        return requests.post(url, data={
-                             'all_members': all_members,
-                             'project_member_ids': project_member_ids,
-                             'subject': subject,
-                             'message': message})
+        r = requests.post(url, data={'all_members': all_members,
+                                     'project_member_ids': project_member_ids,
+                                     'subject': subject,
+                                     'message': message})
+        handle_error(r)
+        return r
+
+
+def handle_error(r):
```

Comment:
This could be improved: 

1. How about also passing the expected `status_code` to make sure that it actually gives the correct reply instead of just having one of the 4? 

2. I think the `return` statements are unnecessary 

3. As the `return` are unnecessary you could just rephrase to `if code != expected_code:` and then do the rest to raise the exception?

Reply:
Sure. Looking into it :)